addons/pos_online_payment_self_order/static/src/pages/order_cart/order_cart.js

```diff
@@ -0,0 +1,67 @@
+/** @odoo-module */
+import { patch } from "@web/core/utils/patch";
+import { OrderCart } from "@pos_self_order/pages/order_cart/order_cart";
+import { _t } from "@web/core/l10n/translation";
+
+patch(OrderCart.prototype, "pos_online_payment_self_order.OrderCart", {
+    get buttonToShow() {
+        if (this.selfOrder.self_order_mode === "each") {
+            return "Pay";
```

Comment:
```suggestion
            return _t("Pay");
```

Should be translated right? Do the same for others.

Reply:
Ok, done